### PR TITLE
Modify Learn Section

### PR DESCRIPTION
**Pull request by salmad3**
Creates categories for all learn docs instead of having them dumped the way they are right now. Some labels left generic to get feedback 

This may change in the future, and serves as way to organize the docs in their current form

### CODE REVIEW

File: docs/learn/learn-registrar.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ---
 id: learn-registrar
 title: Using W3F Registrar
-sidebar_label: How to use W3F Registrar
+sidebar_label: Registrar
```
Comment (profradha) [inline docs/learn/learn-registrar.md]: There are multiple registrars on Polkadot. This article is about W3F Registrar

```suggestion
sidebar_label: W3F Registrar
```

File: docs/learn/learn-security.md [hunk 2]
```diff
@@ -1,7 +1,7 @@
 ---
 id: learn-security
 title: Security of the network
-sidebar_label: Security of the network
+sidebar_label: Security
```
Comment (profradha) [inline docs/learn/learn-security.md]:
```suggestion
sidebar_label: Network Security
```

File: docs/learn/learn-cryptography.md [hunk 3]
```diff
@@ -1,7 +1,7 @@
 ---
 id: learn-cryptography
 title: Cryptography Explainer
-sidebar_label: Cryptography Explainer
+sidebar_label: Cryptography
```
Comment (profradha) [inline docs/learn/learn-cryptography.md]: As they are inside the cryptography dropdown, should this be named "Explainer" instead? "Cryptography Explainer" is also okay.

Having dropdown with that same name as the title feels a little off. Cryptography > Cryptography

File: docs/learn/learn-runtime-upgrades.md [hunk 4]
```diff
@@ -1,7 +1,7 @@
 ---
 id: learn-runtime-upgrades
 title: Runtime Upgrades
-sidebar_label: Runtime Upgrades
+sidebar_label: Upgrades
```
Comment (profradha) [inline docs/learn/learn-runtime-upgrades.md]: Upgrades title is too generic for this article. Runtime upgrades are different from node/client upgrades 

```suggestion
sidebar_label: Runtime Upgrades
```

**Review by DrW3RK — CHANGES_REQUESTED** [review]

File: polkadot-wiki/sidebars.js [hunk 5]
```diff
@@ -26,84 +26,108 @@ module.exports = {
...
-                'learn/learn-teleport',
+                  'learn/learn-accounts',
+                  'learn/learn-account-generation',
+                  'learn/learn-account-restore',
+                  'learn/learn-balance-transfers',
+              ],
+            },
+            {
+              type: 'category',
+              label: 'Assets',
```
Comment (profradha) [inline polkadot-wiki/sidebars.js]: Same title in the dropdown Assets > Assets 

```suggestion
              label: 'Tokens and Assets',
```

File: polkadot-wiki/sidebars.js [hunk 6]
```diff
@@ -26,84 +26,108 @@ module.exports = {
...
+                  'learn/learn-security',
+                  'learn/learn-transaction-fees',
+                  'learn/learn-treasury',
+                  'learn/learn-runtime-upgrades',
+                  'learn/learn-validator',
+              ],
+            },
+            {
+              type: 'category',
+              label: 'Staking',
```
Comment (profradha) [inline polkadot-wiki/sidebars.js:74]: Staking > Staking 

But I do not know what to name it. May be Staking Concepts > Staking? Also, this article is moved from advanced section to basics. We may need a more simpler doc for staking, but that is a discussion for sometime later. I think the placement of this doc in basics is fine.

**Review by DrW3RK — CHANGES_REQUESTED** [review]

File: polkadot-wiki/sidebars.js (hunk 6 above)
Comment (salmad3) [reply polkadot-wiki/sidebars.js:74]: Was thinking the same, we have some docs in the basic section that are perhaps at the same level in some areas, but this doc still dives into details that aren't necessarily basics

Comment (salmad3) [reply polkadot-wiki/sidebars.js:74]: Seeing if Stake > Staking Concepts, Staking FAQ works

**Comment by salmad3** [comment]
thanks @DrW3RK, left some labels generic to get feedback

File: polkadot-wiki/sidebars.js [hunk 7]
```diff
@@ -26,84 +26,108 @@ module.exports = {
...
+                  'learn/learn-registrar',
+                  'learn/learn-runtime-upgrades',
+                  'learn/learn-transaction-fees',
+                  'learn/learn-treasury',
+                  'learn/learn-validator',
+              ],
+            },
+            {
+              type: 'category',
+              label: 'Stake',
```
Comment (profradha) [inline polkadot-wiki/sidebars.js]:
```suggestion
              label: 'Staking',
```

**Review by DrW3RK — APPROVED** [review]
Just a minor fix. 'Staking' is a more appropriate title than 'Stake'

**Pull request merged**